### ISSUE TITLE
How do I get the error information on my jetson of Ricoh Theta Z1.

### ISSUE THREAD

**Issue by RajUpadhyay** [ricohapi/theta-api-specs#17]
Hi,

I am using Ricoh Theta Z1 with my Jetson Xavier NX.
What I want to do right now is get the error information of the Ricoh Theta Z1.

I would really appreciate if you could help me with some sample code to fetch the error codes.
For example, whenever in the error_info.md located in the property folder (attaching the link below)

0x80000000 |          (Board) temperature error 
Whenever this condition occurs, I want to get this error.

Basically a way to fetch these  error or information codes written in the .md files via the api.

Thank You.

System Description:
Ricoh Theta Z1
Jetson Xavier NX
Ubuntu 20.04

Ricoh Theta is connected to the jetson via a type-c cable.
